templates/vars.yml

```diff
@@ -2,30 +2,16 @@ self_version: "v7"
 base_version: "v4"
 
 versions:
-- airflow:    ["1.9"]
-  spark:      ["2.4.5", "2.4.6", "2.4.7"]
-  hadoop:     ["3.1.0"]
-  scala:      ["2.11", "2.12"]
-  python:     ["3.6"]
-  sqlalchemy: ["1.0", "1.1", "1.2", "1.3"]
-
-- airflow:    ["1.9"]
-  spark:      ["3.0.0", "3.0.1"]
-  hadoop:     ["3.2.0"]
-  scala:      ["2.12"]
-  python:     ["3.6"]
-  sqlalchemy: ["1.0", "1.1", "1.2", "1.3"]
-
 - airflow:    ["1.10"]
-  spark:      ["2.4.5"]
-  hadoop:     ["3.1.0"]
-  scala:      ["2.11", "2.12"]
+  spark:      ["3.0.1", "3.0.2"]
+  hadoop:     ["3.2.0", "3.2.1", "3.2.2"]
```

Comment:
Ideally we would You cannot just specify any `3.2.z`, but this follows https://github.com/dsaidgovsg/airflow-pipeline/blob/master/Dockerfile#L7, which means it will be locked to `3.2.0` for most Spark `3.y.z` since the upstream repo builds based on given `pom.xml` template:
https://github.com/apache/spark/blob/v3.1.1/pom.xml#L123

Reply:
Ah i see. Fixed, thanks for the catch!